src/sch/util.jl

```diff
@@ -399,8 +399,16 @@ function estimate_task_costs(state, procs, task, inputs)
     transfer_costs = Dict(proc=>impute_sum([affinity(chunk)[2] for chunk in filter(c->get_parent(processor(c))!=get_parent(proc), chunks)]) for proc in procs)
 
     # Estimate total cost to move data and get task running after currently-scheduled tasks
-    costs = Dict(proc=>state.worker_time_pressure[get_parent(proc).pid][proc]+(tx_cost/tx_rate) for (proc, tx_cost) in transfer_costs)
-
+    costs = Dict(proc-> begin
+                        if proc == state.scheduler
+                            # For non-Chunk, model cost from scheduler to worker
+                            data_size = sum(affinity(input)[2] for input in filter(x -> !(x isa Chunk), inputs))
+                            scheduler_to_worker_cost = (data_size / tx_rate) + state.scheduler_time_pressure
```

Comment:
There is no scheduler-specific time pressure, you can access that with just `state.worker_time_pressure[1]` (as the scheduler runs on worker 1). Therefore, you don't need to deal with that because it's already factored in on the next line.